core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerPool.scala

```diff
@@ -413,6 +413,36 @@ class FunctionPullingContainerPool(
       // Reset the prewarmCreateCount value when do expiration check and backfill prewarm if possible
       prewarmCreateFailedCount.set(0)
       adjustPrewarmedContainer(false, true)
+
+    case StatusQuery =>
```

Comment:
I think we can combine this part with my PR https://github.com/apache/openwhisk/pull/5334

Reply:
That looks great, I would apply it.